lms/templates/main_django.html

```diff
@@ -50,23 +50,15 @@
     {% elif IS_EDX_DOMAIN %}
       {% include "footer-edx-v3.html" %}
     {% else %}
-      {% include "footer.html" %}
+      {% if FEATURES.USE_CUSTOM_THEME %}
```

Comment:
This should be one single `if` block, without a "sub-if".

It should take the form:
```python
if IS_REQUEST_IN_MICROSITE:
    pass
elif IS_EDX_DOMAIN:
    pass
elif FEATURES.USE_CUSTOM_THEME:
    pass
else:
    pass
```